vr-wordle/src/App.js

```diff
@@ -151,20 +156,27 @@ export default function App() {
         />
         <Grabber groupRef={letters} />
         {generateLetters(reset, alphabet, letters)}
-        <Letter position={[2, 1, 0.4]} name="c" />
-        <Letter position={[2, 2, 0.6]} name="h" />
-        <Letter position={[2, 1, 0.7]} name="o" />
-        <Letter position={[-2, 1, 0.6]} name="k" />
-        <Letter position={[-2, 1, 0.4]} name="e" />
-        <Column position={[-1.25, 0, 0.4]} guessIndex={4} setGuess={setGuess} />
-        <Column position={[-0.6, 0, 0.6]} guessIndex={3} setGuess={setGuess} />
-        <Column position={[0, 0, 0.7]} guessIndex={2} setGuess={setGuess} />
-        <Column position={[0.6, 0, 0.6]} guessIndex={1} setGuess={setGuess} />
-        <Column position={[1.2, 0, 0.4]} guessIndex={0} setGuess={setGuess} />
+        <Letter position={[2, 1, 1]} name="w" />
+        <Letter position={[2, 2, 0.6]} name="r" />
+        <Letter position={[2, 3, 0.7]} name="d" />
+        <Letter position={[-2, 1, 0.6]} name="l" />
+        <Letter position={[-2, 2, 1]} name="e" />
+        <Pillars setGuess={setGuess} />
         <Player />
         <Floor />
       </Physics>
-      <Stars />
+      <Environment preset={"night"} />
+      <Sky distance={450000} sunPosition={[0, -1, 0]} azimuth={0.25} />
+      <Stars
+        radius={100}
+        depth={100}
+        count={5000}
+        factor={4}
+        saturation={0}
+        fade
+      />
+      <fog attach="fog" args={["#421700", 0, 100]} />
+      <PositionalAudio url="/wind.mp3" distance={1} autoplay={false} loop />
```

Comment:
Is there a reason autoplay is set to false?

Reply:
ah you're right, that should be set to true!
```suggestion
      <PositionalAudio url="/wind.mp3" distance={1} autoplay={true} loop />
```